scripts/splitEquippableUserJourney.ts

```diff
@@ -213,6 +214,22 @@ async function setupBase(base: SimpleBase, gemAddress: string): Promise<void> {
   console.log("Base is set");
 }
 
+interface AcceptChild {
+  parentId: number;
+  childIndex: number;
+  childAddress: string;
+  childId: number;
+}
+
+function namedAcceptChild(args: AcceptChild): [number, number, string, number] {
+  return [
+    args.parentId,
+    args.childIndex,
+    args.childAddress,
+    args.childId,
+  ];
+}
```

Comment:
This change (as well as all other changes) needs to be reflected in the READMEs.

Reply:
This was a test haha, I commited it by mistake. Will revert.